packages/dd-trace/src/profiling/exporters/agent.js

```diff
@@ -29,46 +26,41 @@ class AgentExporter {
       form.append('tags[]', `${key}:${tags[key]}`)
     }
 
-    eachOfSeries(types, (type, index, callback) => {
-      const profile = profiles[type]
+    let index = 0
+    for (const type of types) {
```

Comment:
Since there is an index, it seems like a regular `for` might be a better fit here.